core/net/src/service.rs

```diff
@@ -34,6 +34,7 @@ pub struct Net;
 
 lazy_static::lazy_static! {
     pub(crate) static ref NET_TYPE: Arc<RwLock<NetType>> = Default::default();
+    pub(crate) static ref NODE_ID: Arc<RwLock<NodeId>> = Default::default();
```

Comment:
Please don't use `lazy_static` if not necassary

Reply:
Did not like it either. Just wanted to avoid repeating gsb calls.
Latest commit deletes them and uses gsb call in REST endpoint instead.